layout/mstab.lua

```diff
@@ -72,7 +72,11 @@ function update_tabbar(
                 c.minimized = true
             end)
         )
-        local client_box = bar.create(c, (idx == top_idx), buttons)
+        local inactive = false
+        if client.focus == awful.client.getmaster() then
+           inactive = true
+        end
```

Comment:
Can't you just do `local inactive = (client.focus == awful.client.getmaster())` instead?